### PR TITLE
do not mount whole FS when checking for files

### PR DESCRIPTION
**Pull request by TomasTomecek**
* this saves so much time; also requires ton of memory if the image is
  beefy

\+

* error messages: grammar
* move test images creation to conftest.py

### CODE REVIEW

File: tests/conftest.py [hunk 1]
```diff
@@ -1,5 +1,49 @@
...
+            backend.ImageClass.build(data_dir, tag=image_name,
+                                     dockerfile=image_data["dockerfile_path"])
+
+
+@pytest.fixture(autouse=True, scope='session')
+def setup_test_session():
+    """ set up environment before testing """
+    for image_name in IMAGES:
+        try:
+            subprocess.check_call(["docker", "image", "inspect", image_name], stdout=subprocess.PIPE)
```
[the remark this answers is not in the document]
Comment (TomasTomecek) [reply tests/conftest.py]: did we agree on 120?

[the remark this answers is not in the document]
Comment (lachmanfrantisek) [reply tests/conftest.py]: Hmmm, no? We've agreed on 100..;-)

-----

Tomorrow, I will synchronize the configs from the tool repo...

[the remark this answers is not in the document]
Comment (TomasTomecek) [reply tests/conftest.py]: okay

**Comment by TomasTomecek** [comment]
Can we merge or do we need to get a blessing from @lachmanfrantisek?

**Pull request merged**